pool/test/matcher_test.ml

```diff
@@ -67,6 +67,7 @@ let create_invitations_model () =
   Test_utils.check_result expected events
 ;;
 
+(* TODO: Remove, this function is unused *)
```

Comment:
open todo

Reply:
Yes about that, why is that test case not run? It is currently failing

Reply:
Created issue